doc/ghcjs.md

```diff
@@ -21,7 +21,13 @@ then you may need to install a different version of `node`. See
 
 ## Example Configurations
 
-### Recent versions of GHCJS, repacked for stack
+### GHCJS repacked for snapsnots lts-8 and lts-9
+
+Please see the [ghcjs-stack-dist repository](https://github.com/matchwood/ghcjs-stack-dist) for `lts-8` and `lts-9` configurations and refer to the [README](https://github.com/matchwood/ghcjs-stack-dist/blob/master/README.md) for common issues.
```

Comment:
I take it we can't just have a table like for the earlier snapshots?

Reply:
We could but I'd prefer people engaged with the README on the ghcjs-stack-dist repo (as there are some important issues flagged up there) and I didn't want to just copy across all that info. If and when we get a better story for ghcjs support I will rewrite the docs and put everything in a table etc.